### PR TITLE
Allow clicking on the name of each entry in order to show the wiki en…

### PR DESCRIPTION
**Pull request by Rahjenaos**
…try. You now have two ways of opening the wiki entry

This should solve https://github.com/elyukai/optolith-client/issues/556
I'd opt for this minimal change due to several points:
* less code change
* prevent showing the Wiki entry (in this case "Zechen") even if clicking on the category ("Körper")  of a talent ("Zechen").
* prevent showing the Wiki entry if clicking on the attributes (e.g. "KK") of a talent.
* doing it this way ("clicking on the name" shows the wiki entry) already solves the main issue with "it is hard to aim for the small 'i' icon"

### CODE REVIEW

File: src/App/Views/Skills/SkillListItem.tsx [hunk 1]
```diff
@@ -80,15 +80,22 @@ const SkillListItem: React.FC<Props> = props => {
     selectForInfo,
   } = props
 
+  const handleSelectForInfo =
+  React.useCallback (
+    () => selectForInfo (id),
+    [ selectForInfo, id ]
+  )
```
Comment (elyukai) [inline src/App/Views/Skills/SkillListItem.tsx:87]: Just a VERY small thing: Would be great if you could indent those lines by two additional spaces. 🙈

File: src/App/Views/Skills/SkillListItem.tsx [hunk 2]
```diff
@@ -80,15 +80,22 @@ const SkillListItem: React.FC<Props> = props => {
...
+
   return (
     <ListItem
       noIncrease={noIncrease}
       recommended={typ}
       unrecommended={untyp}
       insertTopMargin={insertTopMargin}
       active={Maybe.elem (id) (selectedForInfo)}
+
```
Comment (elyukai) [inline src/App/Views/Skills/SkillListItem.tsx:96]: Why is the an empty new line?

Comment (Rahjenaos) [reply src/App/Views/Skills/SkillListItem.tsx]: @elyukai error when removing bad code. thanks for pointing this out. should be gone now

File: src/App/Views/Skills/SkillListItem.tsx (hunk 1 above)
Comment (Rahjenaos) [reply src/App/Views/Skills/SkillListItem.tsx]: @elyukai should hopefully be improved now. github throws a lot of errors at me atm but I hope it went through

**Pull request merged**